apps/api/src/graphql/Project.ts

```diff
@@ -34,8 +34,8 @@ export const ProjectType = objectType({
       description: 'If this project is liked by the current user',
       async resolve(_root, _, ctx) {
         try {
-          const currentUserId = decodeAccessToken(ctx.accessToken);
-          if (!currentUserId) {
+          const currentUserId = decodeAccessToken(ctx?.accessToken);
+          if (currentUserId) {
```

Comment:
could you check this line back to `!currentUserId`

Reply:
done.